test/server/graphql/v2/mutation/ExpenseMutations.test.js

```diff
@@ -753,4 +782,158 @@ describe('server/graphql/v2/mutation/ExpenseMutations', () => {
       });
     });
   });
+
+  describe('processExpense > PAY > with 2FA payouts', () => {
+    const fee = 1.74;
+    let collective, host, collectiveAdmin, hostAdmin, sandbox, expense1, expense2, expense3, expense4, user;
+
+    before(() => {
+      sandbox = sinon.createSandbox();
+      sandbox.stub(paymentProviders.transferwise, 'payExpense').resolves({ quote: { fee } });
```

Comment:
You'll also need to stub `paymentProviders.transferwise.getTemporaryQuote` because of line `server/graphql/v1/mutations/expenses.js:672`:
```suggestion
      sandbox.stub(paymentProviders.transferwise, 'payExpense').resolves({ quote: { fee } });
      sandbox.stub(paymentProviders.transferwise, 'getTemporaryQuote').resolves({ fee });